src/main/scala/org/ergoplatform/http/api/TransactionsApiRoute.scala

```diff
@@ -68,6 +69,7 @@ case class TransactionsApiRoute(readersHolder: ActorRef,
       getUnconfirmedTransactionsR ~
       unconfirmedContainsR ~
       sendTransactionR ~
+      sendTransactionBytesR ~
```

Comment:
Possibly `sendTransactionAsBytes` ... `sendTransactionBytes` looks as if we are sending bytes of a transaction

Reply:
renamed